concepts/basics2/about.md

```diff
@@ -0,0 +1,105 @@
+# Haskell Syntax Basics 2
```

Comment:
Concepts should be be quite granular. There only really should be one basics concept, further concepts should be individual concepts. Currently, this documents lists several. Could you split them up into individual concepts? As such, you might find that there need to be more exercises or perhaps the exercise needs to be tweaked a bit.